src/wp-admin/security.php

```diff
@@ -0,0 +1,194 @@
+<span style="color: #BA2121; font-style: italic"> * @return false|string The resulting page&#39;s hook_suffix, or false if the user</span>
+<span style="color: #BA2121; font-style: italic">                        does not have the 'manage_options' capability.</span>
+<span style="color: #BA2121; font-style: italic"> */</span>
+<span style="color: #008000; font-weight: bold">function</span> <span style="color: #0000FF">add_security_page</span>(
+    <span style="color: #19177C">$page_title</span>,
+    <span style="color: #19177C">$menu_title</span>,
+    <span style="color: #19177C">$menu_slug</span>,
+    <span style="color: #19177C">$function</span> <span style="color: #666666">=</span> <span style="color: #BA2121">&#39;&#39;</span>
+)
+
+</pre>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="meta-box-sortables ui-sortable">
+          <div class="postbox">
+            <h2><span><?php _e( 'Usage' ); ?></span></h2>
+            <div class="inside">
+              <p>You will need to change the logic in your plugin to check for <code>add_security_page()</code>. For example:</p>
+              <!-- HTML generated using hilite.me -->
+              <div style="background: #f8f8f8; overflow:auto;width:auto;border:solid gray;border-width:.1em .1em .1em .8em;padding:.2em .6em;"><pre style="margin: 0; line-height: 125%"><span style="color: #008000; font-weight: bold">if</span> ( <span style="color: #008000">function_exists</span>( <span style="color: #BA2121">&#39;\add_security_page&#39;</span> ) ) {
+    add_security_page(
+        <span style="color: #BA2121">&#39;My Prodigious Plugin&#39;</span>,
+        <span style="color: #BA2121">&#39;My Menu Title&#39;</span>,
+        <span style="color: #BA2121">&#39;my-prodigious-plugin&#39;</span>,
+        <span style="color: #BA2121">&#39;my-output-function&#39;</span>
+    );
+} <span style="color: #008000; font-weight: bold">else</span> {
+    <span style="color: #408080; font-style: italic">// existing code</span>
+}
+</pre>
+              </div>
+              <p>You will also need to change the logic on your settings pages, but that is outside the scope of this guide. However, you should remember that the idea is to <strong>move</strong> security-related settings, <strong>not</strong> to <em>duplicate</em> them.</p>
+            </div>
+            <h2><span><?php _e( 'Security Plugins' ); ?></span></h2>
+            <div class="inside">
+              <p>If your plugin has nothing but security-related settings it may be more useful to provide a summary of the current settings, with links to where they can be changed.</p>
+            </div>
+            <h2><span><?php _e( 'Plugin Action Links'); ?></span></h2>
+            <div class="inside">
+              <p>Many plugins add links to the plugin action links. [As above wrt rephrasing] You can declutter that area for your plugin by using a standard dashicon; for example, the "Settings" link:</p>
```

Comment:
Let's remove this section "Plugin Action Links". It's not directly relevant to the security screen. We can write about this on the forums instead and link to it from the documentation page.

Reply:
Done.